samples/jaxrs-sample/src/main/java/io/servicecomb/samples/jaxrs/client/JaxrsClient.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicecomb.samples.jaxrs.client;
+
+import io.servicecomb.foundation.common.utils.BeanUtils;
+import io.servicecomb.foundation.common.utils.Log4jUtils;
+import io.servicecomb.provider.pojo.RpcReference;
+import io.servicecomb.samples.jaxrs.Hello;
+import io.servicecomb.samples.jaxrs.models.Person;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JaxrsClient {
+
+    @RpcReference(microserviceName = "jaxrs", schemaId = "jaxrsHello")
+    private static Hello hello;
+
+    public static void main(String[] args) throws Exception {
+        init(args);
+        System.out.println(hello.sayHi("Java Chassis"));
```

Comment:
we can't call @RestScheam service with RPC call, must call these method with RestTeplate( REST style api). This can work only because you define only POST methods in your application.